apps/content_types/content_types.py

```diff
@@ -105,13 +129,67 @@ def _set_updated_by(self, doc):
     def _set_created_by(self, doc):
         doc['created_by'] = get_user_id()
 
+    def get_allowed_list(self, schema):
```

Comment:
This is to access some parts of the schema definition for subject and the 'subject' name is not changed (if changed also the mongodb field will change).